app/enterprise/0.33-x/admin-gui/configuration/getting-started.md

```diff
@@ -12,7 +12,7 @@ Start Kong.
 
 `kong start -c kong.conf.default`
 
-> Note: Not all deployments of Kong utilize a configuration file, if this describes you (or you are unsure) please reference the [Kong configuration docs](https://getkong.org/latest/configuration/) in order to implement this step.
+> Note: Not all deployments of Kong utilize a configuration file, if this describes you (or you are unsure) please reference the [Kong configuration docs](/enterprise/0.33-x/developer-portal/configuration/getting-started/) in order to implement this step.
```

Comment:
This new link points from a section of Kong docs concerning the Admin GUI to a section of Kong docs concerning the Developer Portal - I don't think that switch of topic is warranted.

Reply:
I see. When I was looking at this issue, It seemed like if you were browsing the "Enterprise" documentation for (0.33-x), and you clicked a link for configuration items, you should be able to be taken to a page with those configuration items for "Enterprise" not "community edition".

Does that make sense? Would you prefer I remove the line all together if its not needed?

Reply:
Looks like the original intent of this link was to point users to documentation about how to use and verify Kong configuration files. Since this information is common between Kong and Kong Enterprise, it's fine to point to the corresponding version of the Kong doc, in this case, https://docs.konghq.com/0.13.x/configuration/ . (Kong Enterprise 0.33 is based on Kong 0.13.)

In the future it would be good to have a "Kong Enterprise Configuration Properties" page that documents all enterprise-specific configuration properties. That's beyond the scope of this PR.